lib/all-badge-examples.js

```diff
@@ -1455,7 +1452,7 @@ const allBadgeExamples = [
       },
       {
         title: 'GitHub issue title',
-        previewUri: '/github/issues/detail/title/badges/shields/979.svg',
+        exampleUri: '/github/issues/detail/title/badges/shields/979.svg',
```

Comment:
This badge is really wide. Making omitting `previewUri` prevents the badge from displaying and growing the column too wide.